CoreFoundation/Base.subproj/CFInternal.h

```diff
@@ -349,7 +349,7 @@ CF_EXPORT void * __CFConstantStringClassReferencePtr;
 
 #if DEPLOYMENT_RUNTIME_SWIFT
 
-CF_EXPORT void *__CFConstantStringClassReference[];
+CF_EXPORT void *__CFConstantStringClassReference[] asm("_T010Foundation19_NSCFConstantStringCN");
```

Comment:
This won't work on Darwin builds since the module name is `SwiftFoundation` and not `Foundation`